Dutch translation for LLM assisted automation

TheFes opened the pull request.

marcelveldt commented on llm-enhanced-local-assist-blueprint/mass_llm_enhanced_assist_blueprint_nlyaml:
```diff
@@ -0,0 +1,419 @@
+          filter:
+            domain: conversation
+    music_assistant_settings:
+      name: Intstellingen voor afspelen via Music Assistant
```
```suggestion
      name: Instellingen voor afspelen via Music Assistant
```

marcelveldt commented on llm-enhanced-local-assist-blueprint/mass_llm_enhanced_assist_blueprint_nlyaml:
```diff
@@ -0,0 +1,419 @@
+      input:
+        default_player:
+          name: Standaard mediaspeler
+          description: 'De standaar Music Assistant mediaspeler die gebruikt wordt in het
```
```suggestion
          description: 'De standaard Music Assistant mediaspeler die gebruikt wordt in het
```

marcelveldt commented on llm-enhanced-local-assist-blueprint/mass_llm_enhanced_assist_blueprint_nlyaml:
```diff
@@ -0,0 +1,419 @@
+          description: 'Deze instelling bepaalt hoe de "radio_mode" instelling gebruikt
+            moet worden bij het afspelen van de media.
+
+            Als dit op "Altijd" staat, zal Music Assistant altijd media blijven toevoegen
+            nadat het de media uit het commonda gespeeld is.
```
```suggestion
            Als dit op "Altijd" staat, zal Music Assistant proberen 'radio modus' in te schakelen voor de geselecteerde media, wat wil zeggen dat een mix wordt gemaakt van de muziek van jouw selectie en gelijkwaardige nummers.
```

marcelveldt commented on llm-enhanced-local-assist-blueprint/mass_llm_enhanced_assist_blueprint_nlyaml:
```diff
@@ -0,0 +1,419 @@
+            Als dit op "Altijd" staat, zal Music Assistant altijd media blijven toevoegen
+            nadat het de media uit het commonda gespeeld is.
+
+            Als dit op "Nooit" staat, zal Music Assistant stoppen nadat de door de LLM 
+            geselecteerde media gespeeld is.
```
```suggestion
            Als dit op "Nooit" staat, zal Music Assistant alleen de media afspelen die je hebt gevraagd.
```

marcelveldt commented on llm-enhanced-local-assist-blueprint/mass_llm_enhanced_assist_blueprint_nlyaml:
```diff
@@ -0,0 +1,419 @@
+              multiple: false
+          default:
+        play_continuously:
+          name: Play Continuously
```
Eigenlijk zou het fijn zijn om dit consistent Radio modus te noemen.
Ik vraag me nog steeds af of het wel in de scripts moet hoor want we hebben tegenwoordig ook de "dont stop the music" functie die vanzelf radio modus aanzet aan het eind van de queue

TheFes replied:
Ik vind het prima om het er uit te halen hoor, ik heb je op mijn mediaspelers inderdaad ook aan staan

TheFes replied:
Veranderd zoals besproken op Discord

TheFes commented:
@marcelveldt 
I removed the blueprint yaml and instead created a markdown file with the Dutch explanation.
Added a table with translations (only Dutch right now) in the main README which points to this newly created md file.

Can you have look?

marcelveldt reviewed (approved):
Ziet er m.i. goed uit!
